docs/manage/_advanced.rst

```diff
@@ -259,8 +259,8 @@ Restart the server, and your files should be seamlessly detected at the new loca
 
 Alternatively, you can follow these steps in the GUI.
 
-#. Go to **Computer > Advanced System Settings** and press the :guilabel:`Environment Variables` button.
-#. Under **User Variables for...** press the :guilabel:`New...` button.
+#. Go to **Computer > Advanced System Settings** and press the :guilabel:`ENVIRONMENT VARIABLES` button.
+#. Under **User Variables for...** press the :guilabel:`NEW...` button.
```

Comment:
These two buttons are intentionally not capitalized because they refer to Windows GUI:

![2018-11-05_14-34-03](https://user-images.githubusercontent.com/1457929/48001424-dbe4e800-e108-11e8-9998-add3a59b1f32.png)

Reply:
Sure, we can also keep concrete consistency in the abstract representation of a UI label.. but it seems easier to write documentation given that it's the same UPPERCASE regardless.. then we don't have to be sensitive to UK or US Title Cases and all that jazz.

Btw. borrowing some inspiration of the Windows buttons, adding `...` to the button label is a really cool way to indicate that something more follows after clicking it.

Reply:
Don't wanna dig deeper into this, am sure that being consistent with the letter case on the buttons will also work fine :+1: